src/realm/sync/network/network_ssl.hpp

```diff
@@ -131,6 +131,8 @@ class Context {
     /// default certificates for server verification. For OpenSSL,
     /// use_default_verify() corresponds to
     /// SSL_CTX_set_default_verify_paths(SSL_CTX*);
+    ///
+    /// On Windows this also adds a lookup to the system Trusted Root Certification Authorities list.
```

Comment:
Is there a way to know if it failed to find certs for whatever reason so that you could fall back to `m_ssl_context.use_included_certificate_roots();`?

Reply:
The use_ methods are not mutually-exclusive. You can call as many as you like while you’re setting up your context.
Besides, looking up certificates happens during the SSL handshake, way after you’ve set up the context.